CONFIGURE.sh

```diff
@@ -2,12 +2,29 @@
 
 ########################################################################################################################
 ########################################################################################################################
-# this script must be run by root or sudo 
+# this script must be run by root or sudo
 if [[ "$UID" -ne "0" ]] ; then
     echo "ERROR: This script must be run by root or sudo"
     exit
 fi
 
+echo -e "\nWelcome to the Homework Submission Server (HSS) Configuration\n"
```

Comment:
Yes, this looks good overall.  But it might be nice to put some reasonable defaults / sample values for each of these.  As an infrequent user of databases I forget the difference between host vs user vs database name.   Just remind with (with an "e.g., XXX" in the prompt)  what the host vs. url vs. svn path looks like.  An I think we can suggest the hsdbu as the default database_user.  

I've delayed adding prompts for anything but the password to the configuration script because I rerun this too often :(

Maybe the user should fill out a configuration file (json or other) instead of a script with interactive prompts?  I don't know what is the better design.